pkg/controller/daemon/daemon_controller.go

```diff
@@ -1274,7 +1275,8 @@ func (dsc *DaemonSetsController) simulate(newPod *v1.Pod, node *v1.Node, ds *app
 		}
 		// ignore pods that belong to the daemonset when taking into account whether
 		// a daemonset should bind to a node.
-		if metav1.IsControlledBy(pod, ds) {
+		// TODO: replace this with metav1.IsControlledBy() in 1.12
+		if isControlledByDaemonSet(pod, ds.GetUID()) {
```

Comment:
Can you add a benchmark so we can measure the performance when we revert this change?

Reply:
A benchmark of the func? After https://github.com/kubernetes/kubernetes/pull/64915 merged these two are basically the same.